<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="accessibleName">
        <string/>
       </property>
       <attribute name="title">
        <string>Import File</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>911</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Import task set (.csv)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="importNameText">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="importErrorText">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string/>
           </property>
           <property name="backgroundVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Raw task set</string>
       </attribute>
       <widget class="QTableWidget" name="rawTaskSetTable">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>421</width>
          <height>311</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>421</width>
          <height>16777215</height>
         </size>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>120</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>39</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>23</number>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Process name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Period T</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Computation Time T</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Processed task set</string>
       </attribute>
       <widget class="QTableWidget" name="processedTaskSetTable">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>491</width>
          <height>311</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>16777215</height>
         </size>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>120</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>39</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>23</number>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Process name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Period T</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Computation Time T</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>RMS Priority</string>
         </property>
        </column>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>10</y>
          <width>171</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter">
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string>Schedulability Check</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPlainTextEdit" name="utilizationUText"/>
         </item>
         <item row="2" column="0">
          <widget class="QPlainTextEdit" name="plainTextEdit_2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string>Utilization U:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPlainTextEdit" name="plainTextEdit_4">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPlainTextEdit" name="schedulabilityStatusText"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Timeline</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport_File_csv">
   <property name="text">
    <string>Import File (.csv)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
